dawn/renderer/components/peripherals/GenericPeripheral.js

```diff
@@ -1,36 +1,34 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import _ from 'lodash';
 import numeral from 'numeral';
 
 /**
- * A generic peripheral, used when the peripheralType is unknown.
+ * Generic Peripheral for General Case
  */
 const GenericPeripheral = props => (
```

Comment:
Instead of `props`, you can probably just spread out the attributes: `{ id, param, device_name }`.